src/wp-admin/includes/translation-install.php

```diff
@@ -35,23 +34,25 @@ function translations_api( $type, $args = null ) {
 	$res = apply_filters( 'translations_api', false, $type, $args );
 
 	if ( false === $res ) {
-		$url = $http_url = 'https://api.wordpress.org/translations/' . $type . '/1.0/';
-
+		$stats = array(
+			'wp_version' => $cp_version,
```

Comment:
This needs to look a bit different.  You can leave it out here...

Reply:
ok, no changes here